cli/command/trust/sign.go

```diff
@@ -18,19 +18,26 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type signOptions struct {
+	local bool
+}
+
 func newSignCommand(dockerCli command.Cli) *cobra.Command {
+	options := signOptions{}
 	cmd := &cobra.Command{
 		Use:   "sign IMAGE:TAG",
 		Short: "Sign an image",
 		Args:  cli.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runSignImage(dockerCli, args[0])
+			return signImage(dockerCli, args[0], options)
 		},
 	}
+	flags := cmd.Flags()
+	flags.BoolVarP(&options.local, "local", "l", false, "Sign a locally tagged image")
 	return cmd
 }
 
-func runSignImage(cli command.Cli, imageName string) error {
+func signImage(cli command.Cli, imageName string, options signOptions) error {
```

Comment:
I think this should be `runSignImage()` (or `runSign` would be fine too). This is another convention we use in the cli. The `run` prefix refers to the fact this is the function called for `RunE` of the Command.